service/src/main/resources/providers.yml

```diff
@@ -99,6 +99,7 @@ externalcreds:
     era-commons:
       allowedRedirectUriPatterns: [ "https://bvdp-saturn-dev.appspot.com/ecm-callback",
                                     "https://localhost.dsde-dev.broadinstitute.org/",
+                                    "https://localhost.dsde-dev.broadinstitute.org:3000/",
```

Comment:
Note to reviewers, we use both of these urls. When running as a docker image, the app runs on port 80. When running via `npm start`, the app runs on port 3000